server/src/game/player.rs

```diff
@@ -0,0 +1,32 @@
+use actix::{Addr, Actor, Context};
+use log::info;
+
+use crate::engine::{WebSocket, Server};
+
+pub struct Player {
+    pub server_addr: Addr<Server>,
+    pub websocket_addr: Option<Addr<WebSocket>>,
```

Comment:
what is this address supposed to communicate, is it the ws to send state updates to? rename to something that describes the meaning, not the variable type